### PR TITLE
Fixed behat tests

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: src/Sulu/Bundle/TestBundle/Behat/DefaultContext.php [hunk 1]
```diff
@@ -113,6 +113,16 @@ public function iFillTheSelector($selector, $value)
     }
 
     /**
+     * @Given I leave the selector :selector
+     */
+    public function iLeaveTheSelector($selector)
+    {
+        $this->waitForSelector($selector);
+        $this->getSession()->evaluateScript("$('$selector').trigger('change')");
```
Comment (danrot) [inline src/Sulu/Bundle/TestBundle/Behat/DefaultContext.php]: Is this necessary? If you only leave an element, it only triggers `focusout`, not `change`. `change` should be triggered by the component itself, if it is required, shouldn't it?

Comment (wachterjohannes) [inline src/Sulu/Bundle/TestBundle/Behat/DefaultContext.php]: no its not necessary focusout is enough i will remove it!

**Comment by danrot** [comment]
There is still a failing test :confused:

**Comment by wachterjohannes** [comment]
are you sure? hmm thats a completely new failing testcase ... i have restarted it (: hopefully it works then!

**Comment by danrot** [comment]
I've already tried to restart, it's always the same test failing... but seems like I have to fix that now...

**Pull request merged**